src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedAnalyzerReference.xaml

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule
+    Name="ResolvedAnalyzerReference"
+    DisplayName="Resolved Analyzer Reference"
+    PageTemplate="generic"
+    Description="Resolved Analyzer reference properties"
+    xmlns="http://schemas.microsoft.com/build/2009/properties">
+    <Rule.DataSource>
+        <DataSource Persistence="ProjectFile" ItemType="Analyzer" HasConfigurationCondition="False" />
```

Comment:
I don't have a target because Analyzers don't get resolved in any way. The item has the full path. Since you are listening to the JointRuleSource, that produces events from both evaluation and design time builds.